src/tests/test_nlvoorelkaar_demand_org.py

```diff
@@ -0,0 +1,53 @@
+import os
+from unittest import TestCase
+
+import pytest
+import requests_mock
+from bs4 import BeautifulSoup
+
+from models import InitiativeImport
+from platformen.nlvoorelkaar import NLvoorElkaarSource, NLvoorElkaar
+
+
+class TestNLvoorElkaarPlatformSourceDemandOrg(TestCase):
+
+    @requests_mock.Mocker()
+    def setUp(self, request_mock):
```

Comment:
De scheiding is goed, maar er zit nu wel heel veel duplicatie in de setup. Een base class maken voor de setup is wellicht handig. Nou krijg je dan misschien wat ruzie met imports, dan zou je de testcases gewoon even in dezelfde file kunnen houden wat mij betreft.